crates/bevy_ecs/src/query/access.rs

```diff
@@ -59,39 +61,58 @@ impl<T: SparseSetIndex> Access<T> {
         self.writes.contains(index.sparse_set_index())
     }
 
-    /// Sets this `Access` to having read access for all indices.
+    /// Sets this `Access` as having read access to all indexed elements.
     pub fn read_all(&mut self) {
         self.reads_all = true;
     }
 
-    /// Returns true if this `Access` has read access to all indices.
+    /// Returns `true` if this `Access` has read access to all indexed elements.
+    #[inline]
     pub fn reads_all(&self) -> bool {
         self.reads_all
     }
 
-    /// Clears all recorded accesses.
+    /// Sets this `Access` as having write access to all indexed elements.
+    pub fn write_all(&mut self) {
+        self.reads_all = true;
+        self.writes_all = true;
+    }
+
+    /// Returns `true` if this `Access` has write access to all indexed elements.
+    #[inline]
+    pub fn writes_all(&self) -> bool {
```

Comment:
The writes_all looks very similar to write_all. Maybe consider get_writes_all?

Reply:
Thanks! Since `has_read` and `has_write` were already there, I went with `has_read_all` and `has_write_all`.